.idea/encodings.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Putting your .idea directory under versionning can be useful if you are the only developer to work on this project, but it can lead to annoying conflicts if another developer wants to clone your project to work on it, and have a different IDE configuration.